### DISCUSSION TITLE
Exclude third party kernel module for initrd

### DISCUSSION THREAD

**Discussion by ragazenta** [systemd/mkosi#2299] [Q&A] [answered]
Hi,

In my case, I use `8821au.ko` kernel module for TP-Link USB Wireless dongle. I manually compiled it from [here](https://github.com/morrownr/8821au-20210708).
I put it to `mkosi.extras/usr/lib/modules/<version>/kernel/drivers/net/wireless/8821au.ko`. 

After this commit a90850c5b9f8ccc07751e605ec41914c28740d7e, I got error when building default initrd.
```shell
‣  Running modinfo to fetch kernel module dependencies
modinfo: ERROR: Module 8821au not found.
```
I thought I could use `KernelModulesInitrdExclude=8821au`, but still error.

Revert that commit a90850c5b9f8ccc07751e605ec41914c28740d7e solved my issue though.

**Comment by daandemeyer** [comment, marked as answer]
@ragazenta Please give https://github.com/systemd/mkosi/pull/2301 a try and let me know if that fixes the issue
